test/spec-example/shmem_ctx.c

```diff
@@ -69,32 +59,29 @@ int main(void) {
         int ret = shmem_ctx_create(SHMEM_CTX_PRIVATE, &ctx);
 
         if (ret != 0) {
-            printf("%d: Warning, unable to create context (%d)\n", me, ret);
-            ctx = SHMEM_CTX_DEFAULT;
+            printf("%d: Error creating context (%d)\n", me, ret);
+            shmem_global_exit(2);
         }
 
         /* Process tasks on all PEs, starting with the local PE.  After
          * all tasks on a PE are completed, help the next PE. */
         while (pes_done < npes) {
-            long task = shmem_ctx_long_atomic_fetch_inc(ctx, &task_cntr, task_pe);
+            long task = shmem_atomic_fetch_inc(ctx, &task_cntr, task_pe);
             while (task < ntasks) {
                 /* Perform task (task_pe, task) */
                 tasks_done++;
-                task = shmem_ctx_long_atomic_fetch_inc(ctx, &task_cntr, task_pe);
```

Comment:
These changes allow the tests to compile/run when you configure/build with `CC='gcc -std=c99'`. Maybe we should add this as a row in Travis to flag changes like this.

Reply:
Presently, we require all unit tests to support c99. We could also handle this with a make conditional.